R/app_ui.R

```diff
@@ -8,39 +8,45 @@
 #' @export
 
 app_ui <- function(meta, domainData, mapping, standards){
-    ui<-tagList(
-      useShinyjs(),
-      #add_busy_spinner(spin = "fading-circle", position = "bottom-left", timeout=3000),
-       tags$head(
-        tags$style(HTML(readLines( paste(.libPaths(),'safetygraphics','safetyGraphics_app', 'www','index.css', sep="/")))),
-        tags$link(
-          rel = "stylesheet",
-          type = "text/css",
-          href = "https://use.fontawesome.com/releases/v5.8.1/css/all.css"
-        )
-      ),
-      navbarPage(
-        "safetyGraphics",
-        id="nav_id",
-        tabPanel("Home", icon=icon("home"),homeTabUI("home")),
-        navbarMenu('Data',icon=icon("database"),
-          tabPanel("Preview", icon=icon("table"), settingsDataUI("dataSettings", domains=domainData)),
-          tabPanel("Mapping", icon=icon("map"), mappingTabUI("mapping", meta, domainData, mapping, standards)),
-          tabPanel("Filtering", icon=icon("filter"), filterTabUI("filter","dm"))
-        ),
-        navbarMenu('Charts', icon=icon("chart-bar")),
-        tabPanel("Reports", icon=icon("file-alt")),
-        navbarMenu('',icon=icon("cog"),
-          tabPanel(title = "Metadata", settingsMappingUI("metaSettings")),
-          tabPanel(title = "Charts", settingsChartsUI("chartSettings"))
-        ),
-        tags$script(
-          HTML(
+    #read css from pacakge
+    app_css <-  HTML(readLines( paste(.libPaths(),'safetygraphics','safetyGraphics_app', 'www','index.css', sep="/")))
```

Comment:
@jwildfire, there is a typo in safetyGraphics, should be capital G.
```suggestion
    app_css <-  HTML(readLines( paste(.libPaths(),'safetyGraphics','safetyGraphics_app', 'www','index.css', sep="/")))
```

Reply:
Good catch. Not entirely sure the best way to deal with the libPaths issue, so filed #441 to look at it more closely later.